layouts/nvrf/single.html

```diff
@@ -6,6 +6,16 @@
 <link rel="stylesheet" href="{{ .Site.BaseURL }}/assets/styles/nvrf.css">
 {{ partial "header.html" . }}
 <main id="main-content">
+  <div class="bg-blue" id="skip-target">
+    <div class="grid-container">
+        <div class="tablet:grid-col-3 nvrf-hero">
+          <a href="{{ .Site.Home.RelPermalink }}" aria-label="Vote.gov">
+            <img loading="lazy" height="100" src="{{ .Site.BaseURL }}/assets/img/NVRF_hero_lockup.webp" 
```

Comment:
We can remove the `loading` atttribute we want this to load immediately since it's in the heading.